recipes-app/iot2050-event-record/WATCHDOG.md

```diff
@@ -0,0 +1,127 @@
+Due to the fact that the watchdog is non-stoppable, getting a watchdog event always requires opening the watchdog and feeding the watchdog. So `event-record` doesn't contain the watchdog initially.
+
+This README file explains how to get the watchdog reset status and how to inject it into `event-record`.
+
+# How to get the watchdog reset status?
+
+Here is an example to get watchdog reset status, and print it. Let's name it `wdt_example.py`.
```

Comment:
The `wdt_example.py ` below shows how to get the watchdog reset status.

Reply:
ACK and done.